installer/conf/container.conf

```diff
@@ -64,3 +64,17 @@
 <filter oms.container.**>
 	type filter_container
 </filter>
+
+#separate buffer for containers
+<match oms.container.**>
+  type out_oms
+  log_level info
+  buffer_chunk_limit 20m
+  buffer_type file
+  buffer_path %STATE_DIR_WS%/out_oms_docker*.buffer
```

Comment:
if [ -s $OMSAGENTD_DIR/container.conf ] ; then
      update_path $OMSAGENTD_DIR/container.conf
fi
This is the change in copy_omsagent_d_conf() in omsadmin.sh I tested.